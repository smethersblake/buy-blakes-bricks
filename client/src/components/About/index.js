import React from 'react';
import { Link } from 'react-router-dom';

function About() {
    return (
        <div className="container items-center mx-auto p-36">
            <div className='flex mb-4 pb-4'>
                <div className='w-full h-12'>
                    <h1 className='text-6xl text-center font-bold'>About</h1>
                </div>
               
            </div>

            <img src={ require('../assets/Blake.jpg') } />
        
            <p>Hello My name is Blake. Do you have the same problem I have where you can never find the correct
                LEGO brick to finish your newest creation? Well I am here to tell you that on www.buyblakesbricks.com you can
                (you guessed it) buy my LEGO bricks. Every brick on this site has been cleaned and carefully inspected. When 
                you order form www.buyblakesbricks.com each package is shipped with love and care and the hope that LEGO
                brings you as much joy as it has for me though out the years.
                </p>
            <p>This website was created for a group project for the University of Minnesota Coding Bootcamp.
                The LEGO bricks are truly for sale at the price shown please contact me if you are intrested in any item.
            </p>
            
        </div>
        
    )
}

export default About