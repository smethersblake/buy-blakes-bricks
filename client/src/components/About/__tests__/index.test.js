import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import About from "../";

afterEach(cleanup);

describe("About component", () => {
   // Renders
   it("renders", () => {
      render(<About />);
   });

   // Matches Snapshot
   it("matches snapshot", ()=> {
      const { asFragment } = render(<About />);
      expect(asFragment()).toMatchSnapshot();
   });
});
