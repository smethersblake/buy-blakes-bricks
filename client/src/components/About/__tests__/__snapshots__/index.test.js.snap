// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container items-center mx-auto p-36"
  >
    <div
      class="flex mb-4 pb-4"
    >
      <div
        class="w-full h-12"
      >
        <h1
          class="text-6xl text-center font-bold"
        >
          About
        </h1>
      </div>
    </div>
    <p>
      Hello My name is Blake. Do you have the same problem I have where you can never find the correct LEGO brick to finish your newest creation? Well I am here to tell you that on www.buyblakesbricks.com you can (you guessed it) buy my LEGO bricks. Every brick on this site has been cleaned and carefully inspected. When you order form www.buyblakesbricks.com each package is shipped with love and care and the hope that LEGO brings you as much joy as it has for me though out the years.
    </p>
    <p>
      This website was created for a group project for the University of Minnesota Coding Bootcamp. The LEGO bricks are truly for sale at the price shown please contact me if you are intrested in any item.
    </p>
  </div>
</DocumentFragment>
`;
